provider/puppet.rb

```diff
@@ -423,7 +423,7 @@ def emit_nested_object(data)
 
     def emit_nested_object_overrides(data)
       data.clone.merge(
-        field_name: Google::StringUtils.camelize(data[:field], :upper),
```

Comment:
It's this line right here - but no big deal, no need to change it, in my opinion.

Reply:
Sounds good!